src/index.js

```diff
@@ -17,4 +18,8 @@ if (module.hot) {
   module.hot.accept('./components/app', render);
 }
 
+if (process.env.NODE_ENV === 'development') {
```

Comment:
Is this code run in the browser? Not sure what the mechanics are here, although definitely requires a little documentation.